client/db.go

```diff
@@ -109,7 +110,16 @@ type Store interface {
 	//
 	// Field [FieldKind] values may be provided in either their raw integer form, or as string as per
 	// [FieldKindStringToEnumMapping].
-	PatchSchema(context.Context, string) error
+	PatchSchema(context.Context, string, bool) error
```

Comment:
What are your thoughts on inverting the boolean usage here? 

I think it'd be nice when using the http or cli interfaces that the lack of the boolean would indicate the original functionality of setting the default. 

Requiring an extra `--set-default-schema` cli flag or `SetDefaultSchema` http property would break backwards compatibility.

Reply:
I want switching schema versions to be an explicit choice - it is a major, and likely breaking, change to the user's database and I don't want them to do it accidentally via parameter omission.